mlflow/cli/traces.py

```diff
@@ -692,9 +696,9 @@ def get_assessment(trace_id: str, assessment_id: str) -> None:
 def update_assessment(
     trace_id: str,
     assessment_id: str,
-    value: str | None,
-    rationale: str | None,
-    metadata: str | None,
+    value: str | None = None,
+    rationale: str | None = None,
+    metadata: str | None = None,
 ) -> None:
     """
     Update an existing assessment.
```

Comment:
(please click the down arrow icon before reading this comment). CLI usage examples might become noise for coding agents.

Reply:
What I observed is that agents uses `experiment-id` instead `experiment_id` even when the param spec is `experiment_id`.

Reply:
Interesting. It's working fine now.